goconvey.go

```diff
@@ -63,18 +62,20 @@ func folders() {
 		log.Fatal(err)
 	}
 
-	tmpDest := filepath.Join(os.TempDir(), fmt.Sprintf("/goconvey-%v", currentUser.Uid))
+	tmpDest := filepath.Join(os.TempDir(), fmt.Sprintf("goconvey-%v", currentUser.Uid))
 
 	mkdir := func(folder string) {
 		if err := os.MkdirAll(folder, 0700); err != nil {
-			log.Fatal(err)
+			if !os.IsExist(err) {
+				log.Fatal(err)
+			}
 		}
 	}
 
 	cleanupAndMkdir := func(folder string) {
 		if stat, err := os.Stat(folder); err == nil && stat != nil {
 			if !stat.IsDir() {
-				if err = os.Remove(folder); err != nil {
+				if err = os.RemoveAll(folder); err != nil {
```

Comment:
Not sure about this... probably will go for just os.Remove, because it'll delete all the symlink'ed content.

Reply:
Made another commit...
@riannucci  Should I squash it all and force push for convenience ?